src/os/filesystem.go

```diff
@@ -46,6 +46,9 @@ type FileHandle interface {
 	// Read reads up to len(b) bytes from the file.
 	Read(b []byte) (n int, err error)
 
+	// Pread reads up to len(b) bytes from the file starting at the given absolute offset
+	Pread(b []byte, offset int64) (n int, err error)
```

Comment:
Why call this `Pread`? I think calling it `ReadAt` is nicer to match [`io.ReaderAt`](https://pkg.go.dev/io#ReaderAt) (in fact, you might want to embed `io.ReaderAt` here instead).

Reply:
I was just following upstream's os.File ReadAt's example, but I guess that's not a very good reason.

Reply:
It has been ReadAtified.